lang/rust/avro/src/lib.rs

```diff
@@ -841,6 +841,7 @@
 //!
 
 mod bigdecimal;
+mod bytes;
```

Comment:
I recall you asked for some re-export related to bytes in rsgen-avro.
Do you still need it ?

Reply:
Indeed, that was my first guess. Then I realized that there is the ambiguous serialization to either `Value::Bytes` or `Value::Fixed` that couldn't be solved by just using `serde_bytes`. Therefore, I ended up using your approach of having a thread local variable to guide the (de)serlializer. With this final approach only `serde_avro_{bytes,fixed,slice}` need to be re-exported, as it is the case in the PR.